src/Context/Context.js

```diff
@@ -45,7 +46,7 @@ const Context = ({ children }) => {
         fetchData()
     }, [])
     return (
-        <StateContext.Provider value={{ categories: state.categories, videos: state.videos, showtoast: state.showtoast, sortByLatest: state.sortByLatest, selectedCategory: state.selectedCategory,playlists:state.playlists,showPlaylistModal:state.showPlaylistModal,currentVideo:currentVideo,setCurrentVideo:setCurrentVideo, watchlater:state.watchlater,dispatch }}>{children}</StateContext.Provider>
+        <StateContext.Provider value={{ categories: state.categories, videos: state.videos, showtoast: state.showtoast, sortByLatest: state.sortByLatest, selectedCategory: state.selectedCategory,playlists:state.playlists,showPlaylistModal:state.showPlaylistModal,currentVideo:currentVideo,setCurrentVideo:setCurrentVideo,watchlater:state.watchlater,likedVideos:state.likedVideos,dispatch }}>{children}</StateContext.Provider>
```

Comment:
I also suggest to add some formatting to the code and configure to format on save